Dockerfile

```diff
@@ -1,3 +1,51 @@
-FROM drydock-prod.workiva.net/workiva/dart_build_image:1 as build
+####################
+##   Dart Stage   ##
+####################
+FROM drydock-prod.workiva.net/workiva/dart2_base_image:1 as build
```

Comment:
Switched images here because the build image was doing base checks like analyzer and format on a language version older than what dart considers "stable". For this repo we want latest stable because is an OSS community package, not a workiva-specific one.